### PR TITLE
Update to Capella 6.1

### PR DESCRIPTION
**Pull request by tguiu**
do not merge before sonar issue is solved

### CODE REVIEW

File: plugins/org.polarsys.capella.scenario.editor.doc/html/Release Notes/Whatsnew.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 	<head>
 		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
-		<title>What's New</title>
```
Comment (mchoubby) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/Release Notes/Whatsnew.html:5]: keep What's New

File: plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html [hunk 2]
```diff
@@ -270,7 +270,7 @@ <h3 id="Combined_Fragments">Combined Fragments</h3>
 				</td>
 				<td>
 					<b>alt</b> "condition A" 
-					<b>over</b> "timeline1" "timeline2"… {
+					<b>over</b> "timeline1" "timeline2"� {
```
Comment (mchoubby) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html]: ?

File: plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html [hunk 3]
```diff
@@ -288,7 +288,7 @@ <h3 id="Combined_Fragments">Combined Fragments</h3>
 				</td>
 				<td>
 					<b>par</b> "condition A" 
-					<b>over</b> "timeline1" "timeline2"… {
+					<b>over</b> "timeline1" "timeline2"� {
```
Comment (mchoubby) [inline plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html]: same

File: plugins/org.polarsys.capella.scenario.editor.doc/html/Release Notes/Whatsnew.html (hunk 1 above)
Comment (tguiu) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/Release Notes/Whatsnew.html:5]: unfornatelty the title comes from the file name. I guess there was an edition by hand

File: plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html (hunk 2 above)
Comment (tguiu) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html]: This special char does not seem to be supported on my platform.

File: plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html (hunk 3 above)
Comment (tguiu) [reply plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html]: same !

**Review by mchoubby — APPROVED** [review]

**Review by pdulth — APPROVED** [review]

**Pull request merged**